tools/gradle-plugin/src/main/java/io/smallrye/openapi/gradleplugin/Configs.java

```diff
@@ -50,6 +51,7 @@ class Configs {
     final Property<String> infoLicenseName;
     final Property<String> infoLicenseUrl;
     final Property<OperationIdStrategy> operationIdStrategy;
+    final Property<DuplicateOperationIdBehavior> duplicateOperationIdBehavior;
```

Comment:
Please also add to `io.smallrye.openapi.mavenplugin.MavenConfig` in the `maven-plugin` module.

Reply:
Done.